bin/picca_deltas.py

```diff
@@ -296,27 +296,29 @@ def cont_fit(data):
         log.write("Found {} DLAs in forests\n".format(nb_dla_in_forest))
 
     ## cuts
-    for p in list(data.keys()):
+    log.write("INFO: Sample has {} forests\n".format(sp.sum([len(p) for p in data.values()])))
+    for p in data.keys():
         l = []
         for d in data[p]:
             if not hasattr(d,'ll') or len(d.ll) < args.npix_min:
-                log.write("{} forest too short\n".format(d.thid))
+                log.write("INFO: Rejected {} due to forest too short\n".format(d.thid))
                 continue
 
             if isnan((d.fl*d.iv).sum()):
-                log.write("{} nan found\n".format(d.thid))
+                log.write("INFO: Rejected {} due to nan found\n".format(d.thid))
                 continue
 
             if(args.use_constant_weight and (d.fl.mean()<=0.0 or d.mean_SNR<=1.0 )):
-                log.write("{} negative mean of too low SNR found\n".format(d.thid))
+                log.write("INFO: Rejected {} due to negative mean of too low SNR found\n".format(d.thid))
                 continue
 
             l.append(d)
             log.write("{} {}-{}-{} accepted\n".format(d.thid,
-                d.plate,d.mjd,d.thid))
+                d.plate,d.mjd,d.fid))
         data[p][:] = l
         if len(data[p])==0:
             del data[p]
+    log.write("INFO: Sample has {} forests\n".format(sp.sum([len(p) for p in data.values()])))
```

Comment:
Maybe something like "Accepted sample has..." (just to clarify the difference to the earlier line)

Reply:
Done